client/my-sites/promote-post/components/campaigns-empty/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { translate } from 'i18n-calypso';
+import megaphoneIllustration from 'calypso/assets/images/customer-home/illustration--megaphone.svg';
+import EmptyContent from 'calypso/components/empty-content';
+import './style.scss';
+
+export default function CampaignsEmpty() {
+	return (
+		<div className="campaigns-empty__container">
+			<EmptyContent
+				className="campaigns-empty"
+				title={ translate( 'Looking for a campaign?' ) }
+				line={ 'There are no running campaigns now.' }
+				action={ 'Start a campaign' }
+				actionURL={ 'todo' }
+				secondaryAction={ 'Learn more' }
```

Comment:
I guess we miss "translate" here, but this feature might have design changes so I think it's ok to postpone them